landlab/components/overland_flow/generate_overland_flow_Bates.py

```diff
@@ -101,61 +99,78 @@ def __init__(
         mannings_n=0.03,
         g=9.81,
```

Comment:
`g=scipy.constants.g` just for consistency?

Reply:
Fixed (here and elsewhere). For some tests this now means specifying that g=9.81